crates/cxx-qt-build/src/lib.rs

```diff
@@ -90,7 +84,18 @@ impl GeneratedCpp {
                             rust_file_path.display());
                     }
 
-                    file_ident = m.ident.to_string().to_case(Case::Snake);
+                    // Match upstream where they use the file name as the ident
+                    //
+                    // TODO: what happens if there are folders?
```

Comment:
need to check what CXX does with folders

Reply:
Yes, if you have a `CRATE/src/folder/binding.rs` CXX ends up building it as `target/cxxbridge/CRATE/src/folder/binding.rs.{cc,h}` so we should follow this.

Reply:
This breaks a lot of things as cxx-qt-lib assumes everything ends up without folders and our other tests are considering that the `src` folder would be in the path too.

Maybe I should do this as a separate change with the cxx_file_stem becoming mandatory as well ?

Reply:
Split into #855